gramps/gui/widgets/cellrenderertextedit.py

```diff
@@ -0,0 +1,63 @@
+#
+# Gramps - a GTK+/GNOME based genealogy program
+#
+# Copyright (C) 2015-2016 Gramps Development Team
+# Copyright (C) 2016      Nick Hall

```

Comment:
Add your own copyright.  I'm not sure my copyright is needed.  Did you copy this file from somewhere?